### PR TITLE
[DEV-16] feat -- added stuffs

### PR DESCRIPTION
**Pull request by NielUp**

### CODE REVIEW

**Review by DavLac — COMMENTED** [review]
Please add a content lazy boy

**Comment by NielUp** [comment]
only because you ask so nicely

**Comment by Ohkwari** [comment]
You didnt specify currency! can you defince a currency, my cryptofarm just crashed

File: Cryptoscam.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+text in here
+
+no scam, real money easy here
+
+gib moneh plis
```
Comment (Ohkwari) [inline Cryptoscam.txt:5]: You didnt specify currency! Please add a currency, because my cryptofarm just blew up.

**Review by Ohkwari — CHANGES_REQUESTED** [review]

File: Cryptoscam.txt (hunk 1 above)
Comment (NielUp) [reply Cryptoscam.txt:5]: DONE!

**Pull request merged**